### DISCUSSION TITLE
Failure checking out code on Orion

### DISCUSSION THREAD

**Discussion by mrinalbiswas** [hafs-community/HAFS#184] [General]
ISSUE: 
This is to report failure of checking out code on Orion. The error message when you clone the code is as follows:

Submodule path 'sorc/hafs_forecast.fd/FV3/ccpp/physics': checked out '43159129954280bfb6fa43f24efb815f66338115'
Submodule 'physics/rte-rrtmgp' (https://github.com/earth-system-radiation/rte-rrtmgp) registered for path 'physics/rte-rrtmgp'
Cloning into 'physics/rte-rrtmgp'...
....
Resolving deltas: 100% (3424/3424), done.
**fatal: reference is not a tree: 7f01618c92409658bddd3afa9acb004c608f6a0d
Unable to checkout '7f01618c92409658bddd3afa9acb004c608f6a0d' in submodule path 'sorc/hafs_forecast.fd/FV3/ccpp/physics/physics/rte-rrtmgp'**
Cloning into 'upp'...

SOLUTION: 
Upgrade the GIT version to 2.21.0
You can do this by:
module load git/2.21.0

**Comment by BinLiu-NOAA** [comment]
Issue addressed/obsolete.
